scripts/deploy/contracts/InvestorVesting.ts

```diff
@@ -6,39 +6,151 @@ import hre, { ethers } from "hardhat";
 
 export const contractNames = () => ["InvestorVesting"];
 
-const tokenAddress = contracts[process.env.HARDHAT_NETWORK?.toLowerCase()].REMI;
-const startDate = 1660274000;
+const tokenAddress = contracts[process.env.HARDHAT_NETWORK].REMI;
+// Ensure this is a moment in the future!
+
+const startDate = 1662127200;
 
 const vestingSeed = [
   {
-    beneficiary: "0x042BC2D085c0584Bd56D62C170C4679e1ee9FC45",
-    vestingAmount: ethers.BigNumber.from("11150000"),
-    duration: 9000,
+    beneficiary: "0x470D968F0d27075F3Db0f28d2C2a5a6EEbaD9E65",
+    vestingAmount: ethers.utils.parseEther("750000000"),
+    duration: 63072000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("300000000"),
+    initialClaimed: false,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0xb5408c2977daa3E371279b72bD598000db84d12a",
+    vestingAmount: ethers.utils.parseEther("1115000000"),
+    duration: 63072000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("0"),
+    initialClaimed: true,
+    claimStartTime: startDate + 15552000
+  },
+  {
+    beneficiary: "0x6A378aFDd6Fc4387791992aA893FC6F9c7EC258b",
+    vestingAmount: ethers.utils.parseEther("500000000"),
+    duration: 63072000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("50000000"),
+    initialClaimed: false,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0xB5AA6A2BA240Bd900005F028716C837f2faaD95D",
+    vestingAmount: ethers.utils.parseEther("500000000"),
+    duration: 63072000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("50000000"),
+    initialClaimed: false,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0xACf1B2B00801Dd725Ee5393dd1E24ea74dDfBCEB",
+    vestingAmount: ethers.utils.parseEther("3000000000"),
+    duration: 157680000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("0"),
+    initialClaimed: true,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0x6b441cC275D3088A62dD12609acEa6f446C645b6",
+    vestingAmount: ethers.utils.parseEther("2000000000"),
+    duration: 157680000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("0"),
+    initialClaimed: true,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0xD8C128747183AF6EbdA860A2ea9B77b3933c25b4",
+    vestingAmount: ethers.utils.parseEther("500000000"),
+    duration: 157680000,
```

Comment:
Should include cliff
```suggestion
    duration: 189216000,
```